core/shared/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -51,7 +50,7 @@ object Renderer {
     private val dateFormat =
       DateTimeFormatter
         .ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz")
-        .withLocale(Locale.US)
+        // .withLocale(Locale.US) // Causes problems with Scala.js
```

Comment:
Yikes.  How does scala.js do locales?  I think this is pretty important on machines with Unamerican default locales.

Reply:
I know how to fix this now! See https://github.com/http4s/http4s/pull/4938#discussion_r683103926

Reply:
Basically, Scala.js ships with virtually no Locales. Whatever you want, you got to build it yourself, or find someone who has.

Reply:
Then what is this going to do on a machine whose default language is not English?  We can't have the weekday and month rendered in an arbitrary language and still have valid HTTP dates.

Reply:
Oh, sorry, I read the grim second comment and not the hopeful comment above it.

Reply:
Fixed :)